packages/server-wallet/src/models/ledger-requests.ts

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash';
+import {Transaction} from 'objection';
+
+import {Bytes32} from '../type-aliases';
+
+// TODO: Is this the same as an objective?
+export type LedgerRequestStatus =
+  | 'pending' // Request added to DB to be approved or rejected by queue
+  // | 'rejected' // Rejected due to lack of available funds TODO: Implement
+  | 'succeeded'; // Ledger update became supported and thus request succeeded
+// | 'failed'; // Rejected for an unexpected reason (some error occurred) TODO: Implement
+
+export type LedgerRequestType = {
+  ledgerChannelId: Bytes32;
+  fundingChannelId: Bytes32;
+  status: LedgerRequestStatus;
+};
+
+// FIXME: (SQL Ledger Models)
+export class LedgerRequests {
+  // Store requests for the ledger's funds
+  private requests: {
+    [fundingChannelId: string]: LedgerRequestType;
```

Comment:
this it throwing me: I don't think it matters _here_, but I would have expected this to be indexed by _ledgerChannelId_ ?

Reply:
I thought it made more sense to index on the channel requesting usage of the ledger since there will only ever be one request on a ledger for a channel but there may be many requests for the same ledger

Reply:
ok. Is there a better name than `fundingChannelId`? Maybe `channelToBeFunded`?

Reply:
This name is not really relevant: this is just a type definition of a type whose keys are arbitrary strings.